src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,113 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it('should return invalid message for incorrect email', () => {
+    const result = validateRegisterForm('test@com', 'P@ssword1!');
+
+    expect(result).toEqual({
+      code: 422, message: 'Email is invalid.',
+    });
+  });
+
+  it(`should return invalid message for both incorrect email
+    and password`, () => {
+    const result = validateRegisterForm('testcom', 'sswo');
+
+    expect(result).toEqual({
+      code: 500, message: 'Password and email are invalid.',
```

Comment:
The expected status code for this test case should be `500` instead of `422` as per the task description. Please update this to ensure it aligns with the requirements.